packages/ui/src/App.tsx

```diff
@@ -30,9 +28,13 @@ const App = () => {
   useEffect(() => {
     (async () => {
       setLoading(true);
-      // TODO: error handling
-      const { settings } = await getSignInExperienceSettings();
+      const settings = await getSignInExperienceSettings();
+
+      // Note: i18n must be initialized ahead of global experience settings
+      await initI18n(settings.languageInfo);
```

Comment:
nit: How about just `language` instead of `languageInfo`?

Reply:
`language` is easy to conflict with other namings, e.g. a single language field.

e.g.
- `Language`: existing enum 
- language: one language
- languageInfo: more than language

Reply:
that is aligned with the current API scheme.  Let's keep it this way for now